README.md

```diff
@@ -206,15 +216,49 @@ $apiResponse = $rcsdk->platform()->post('/subscription', array(
 
 When webhook subscription is created, it will send a request with `validation-token` in headers to webhook address. Webhook address should return a success request with `validation-token` in headers to finish webhook register.
 
+## WebSocket Subscriptions
+
+```php
+use RingCentral\SDK\WebSocket\WebSocket;
+use RingCentral\SDK\WebSocket\Subscription;
+use RingCentral\SDK\WebSocket\Events\NotificationEvent;
+
+// connect websocket
+$websocket = $rcsdk->initWebSocket();
+$websocket->addListener(WebSocket::EVENT_READY, function (SuccessEvent $e) {
+    print 'Websocket Ready' . PHP_EOL;
+    print 'Connection Details' . print_r($e->apiResponse()->body(), true) . PHP_EOL;
+});
+$websocket->addListener(WebSocket::EVENT_ERROR, function (ErrorEvent $e) {
+    print 'Websocket Error' . PHP_EOL;
+});
+$websocket->connect();
+
+// create subscription
+$subscription = $rcsdk->createSubscription();
+$subscription->addEvents(array(
+    '/restapi/v1.0/account/~/extension/~/presence',
+    '/restapi/v1.0/account/~/extension/~/message-store/instant?type=SMS'
+));
+$subscription->addListener(Subscription::EVENT_NOTIFICATION, function (NotificationEvent $e) {
+    print 'Notification ' . print_r($e->payload(), true) . PHP_EOL;
+});
+$subscription->register();
+```
+
+We need to create websocket connection before creating subscription. When websocket connection get error, need to re-created websocket and subscription manually.
```

Comment:
Is there a ws disconnected event? I am not sure it will trigger EVENT_ERROR when network offline, maybe it triggers something like "EVENT_DISCONNECTED".

Reply:
Yes, we also have EVENT_CLOSE event.